ktor-utils/src/io/ktor/util/date/ParseDate.kt

```diff
@@ -0,0 +1,18 @@
+package io.ktor.util.date
+
+fun parseDateRFC850(rfc850: String) : GMTDate{
+
+    return try {
+        val subDate = rfc850.subSequence(rfc850.indexOf(',')+2, rfc850.length)
+        val parts = subDate.split(' ')
+        val date = parts[0].split('-')
+        val time = parts[1].split(':')
+
+        GMTDate(time[2].toInt(),time[1].toInt(),time[0].toInt(), date[0].toInt(), Month.from(date[1]), yearCommonEra( date[2].toInt() ) )
+    }catch (ex:Exception){
+        GMTDate.START
```

Comment:
Maybe we should throw `FailToParseDateException` here?

Reply:
Or maybe null could do the job

Reply:
`null` is for missing